cli/src/cmd/init.js

```diff
@@ -33,22 +33,80 @@ export async function init() {
     });
 
     if (await folderExists(projectName)) {
-      spinner.fail(
-        `Target directory "${targetDir}" already exists. Remove it or choose a different name.`
+      throw Error(
+        `Target directory "${projectName}" already exists. Remove it or choose a different name.`
       );
-      process.exit(1);
     }
 
+    const INIT_BASIC = 'basic';
+    const INIT_ADVANCED = 'advanced';
+    const { initType } = await spinner.prompt({
+      type: 'select',
+      name: 'initType',
+      message: 'What kind of project do you want to init?',
+      choices: [
+        {
+          title: 'hello-world',
+          value: INIT_BASIC,
+          selected: true,
+          description: 'iapp quick start',
+        },
+        {
+          title: 'advanced',
+          value: INIT_ADVANCED,
+          description: 'more configuration options for advanced users',
+        },
+      ],
+    });
+
+    const {
+      useArgs = true,
+      useProtectedData = false,
+      useInputFile = false,
+      useRequesterSecret = false,
+      useAppSecret = false,
+    } = initType === INIT_ADVANCED
+      ? await spinner.prompt([
+          {
+            type: 'confirm',
+            name: 'useArgs',
+            message: 'Would you like to use positional args inside your iApp?',
+            initial: true,
+          },
+          {
+            type: 'confirm',
+            name: 'useProtectedData',
+            message:
+              'Would you like to access a protected data inside your iApp?',
+            initial: false,
+          },
+          {
+            type: 'confirm',
+            name: 'useInputFile',
+            message:
+              'Would you like to use input files inside your iApp? (input files are files downloaded from the internet)',
```

Comment:
looking for suggestions to improve understandability

Reply:
`Do you want to use input files in your iApp? (Note: Input files are typically downloaded from the internet)`